node/src/database/db_migrations.rs

```diff
@@ -2031,4 +2210,159 @@ mod tests {
+            assert_eq!(row.get::<usize, i64>(2).unwrap(), 9223315252854775808); //low bytes
+            assert_eq!(row.get::<usize, i64>(3).unwrap(), 22222);
+            Ok(())
+        });
+        assert_values_still_there_plus_pair_of_integer_columns(&*conn, "pending_payable", |row| {
+            assert_eq!(
+                row.get::<usize, String>(0).unwrap(),
+                "0xb5c8bd9430b6cc87a0e2fe110ece6bf527fa4f222a4bc8cd032f768fc5219838".to_string()
+            );
+            assert_eq!(row.get::<usize, i64>(1).unwrap(), 0); //high bytes
+            assert_eq!(row.get::<usize, i64>(2).unwrap(), 9123000000000); //low bytes
+            assert_eq!(row.get::<usize, i64>(3).unwrap(), 33333);
+            assert_eq!(row.get::<usize, i64>(4).unwrap(), 1);
+            assert_eq!(row.get::<usize, Option<String>>(5).unwrap(), None);
+            Ok(())
+        });
+        let (rate_pack, encrypted) = retrieve_config_row(&*conn, "rate_pack");
+        assert_ne!(old_rate_pack_in_gwei, DEFAULT_RATE_PACK.to_string());
+        assert_eq!(rate_pack, Some(DEFAULT_RATE_PACK.to_string()));
+        assert_eq!(encrypted, false);
+    }
+
+    #[test]
+    fn migration_from_6_to_7_without_any_data() {
+        init_test_logging();
+        let dir_path = ensure_node_home_directory_exists(
+            "db_migrations",
+            "migration_from_6_to_7_without_any_data",
+        );
+        let db_path = dir_path.join(DATABASE_FILE);
+        let _ = bring_db_0_back_to_life_and_return_connection(&db_path);
+        let subject = DbInitializerReal::default();
+        let conn = subject
+            .initialize_to_version(
+                &dir_path,
+                6,
+                true,
+                DbInitializationConfig::create_or_migrate(make_external_migration_parameters()),
+            )
+            .unwrap();
+        let mut subject = DbMigratorReal::new(make_external_migration_parameters());
+        subject.logger = Logger::new("migration_from_6_to_7_without_any_data");
+
+        subject.migrate_database(6, 7, conn).unwrap();
+
+        let test_log_handler = TestLogHandler::new();
+        ["payable", "receivable", "pending_payable"]
+            .iter()
+            .for_each(|table_name| {
+                test_log_handler.exists_log_containing(&format!("DEBUG: migration_from_6_to_7_without_any_data: Migration from 6 to 7: no data to migrate in {}", table_name));
+            })
+    }
+
+    fn insert_value(conn: &dyn ConnectionWrapper, insert_stm: &str) {
+        let mut statement = conn.prepare(insert_stm).unwrap();
+        statement.execute([]).unwrap();
+    }
+
+    fn assert_values_still_there_plus_pair_of_integer_columns(
+        conn: &dyn ConnectionWrapper,
+        table: &str,
+        expected_typed_values: fn(&Row) -> rusqlite::Result<()>,
+    ) {
+        let mut statement = conn.prepare(&format!("select * from {}", table)).unwrap();
```

Comment:
I'd like to see a list of column names rather than *, just because SQLite does not guarantee column order when * is used. Also, you didn't like the name of this helper function.